backend/src/main/java/com/woowacourse/levellog/prequestion/domain/PreQuestion.java

```diff
@@ -0,0 +1,64 @@
+package com.woowacourse.levellog.prequestion.domain;
+
+import com.woowacourse.levellog.common.domain.BaseEntity;
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class PreQuestion extends BaseEntity {
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_levellog"))
+    private Levellog levellog;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_from_member"))
+    private Member from;
```

Comment:
```suggestion
    private Member author;
```
from 보다는 author 어떠신가요?

Reply:
좋습니당